pkgs/by-name/po/portmaster/package.nix

```
@@ -0,0 +1,52 @@
+{
+  lib,
+  buildGoModule,
+  fetchFromGitHub,
+  iptables,
+}:
+buildGoModule rec {
+  pname = "portmaster";
+  version = "1.6.18";
+  CGO_ENABLED = 0;
+
+  ldflags =
+    let
+      BUILD_PATH = "github.com/safing/portbase/info";
+    in
+    [
+      "-X ${BUILD_PATH}.commit=v${version}"
+      "-X ${BUILD_PATH}.buildOptions="
+      "-X ${BUILD_PATH}.buildUser=nixpkgs"
+      "-X ${BUILD_PATH}.buildHost=hydra"
+      "-X ${BUILD_PATH}.buildDate=01.08.2024"
+      "-X ${BUILD_PATH}.buildSource=${src.gitRepoUrl}"
+    ];
+
+  src = fetchFromGitHub {
+    owner = "safing";
+    repo = pname;
+    rev = "v${version}";
+    hash = "sha256-K/HEDVWgjW//m+CqIiL+xgKRObNMOtjY1Z8myTkDXSw=";
+  };
+
+  vendorHash = "sha256-/sxjCSPhsZZwQv7w1bKiBoBnS7jozJIbyu8S64TOe4Q=";
+
+  runtimeDependencies = [ iptables ];
+
+  postPatch = ''
+    substituteInPlace service/updates/main.go --replace 'DisableSoftwareAutoUpdate = false' 'DisableSoftwareAutoUpdate = true'
+  '';
+
+  # integration tests require root access
+  doCheck = false;
+
+  meta = with lib; {
+    description = "A free and open-source application firewall that does the heavy lifting for you";
```

```suggestion
    description = "Free and open-source application firewall that does the heavy lifting for you";
```

*`meta.description` must: [...] Not start with the definite or an indefinite article.*

> https://github.com/NixOS/nixpkgs/tree/master/pkgs#meta-attributes

---

Done :3